pages/common/docker-login.md

```diff
@@ -0,0 +1,20 @@
+# docker login
+
+> Log into a docker registry.
+> More information: <https://docs.docker.com/engine/reference/commandline/login/>.
+
+- Log into registry (user will be prompted for a username and password):
```

Comment:
How about
```suggestion
- Interactively log into a registry:
```
Something similar also would apply to the second example. Also, in which registry does it log into. I don't know Docker, but is it something like the registry in the current directory?